source/hassio/snapshots.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: "Hass.io snapshots"
+description: "Back up your Home Assistant configuration in Hass.io."
+date: 2019-01-09 00:01:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+To back up your Home Assistant configuration in Hass.io, you can use the 'Snapshot' function to create backups.
+
+## {% linkable_title Ways to interact %}
+
+Besides the interface available via the Hass.io panel in Home Assistant, there are other ways to use this feature.
+
+### {% linkable_title Command line %}
+
+When using [the command line](/hassio/commandline/), you can use the `hassio snapshots` command to list snapshots, create new ones, restore others, and more.
+
+### {% linkable_title Services %}
+
+Can be used in automations.
+
+### {% linkable_title Automation example %}
```

Comment:
Pherhaps you can use this as [example script](https://github.com/klaasnicolaas/Smarthome-homeassistant-config/blob/master/automations/system/auto_backup.yaml)